### PR TITLE
match funcs in code_a5dd0_len_114e0

### PR DESCRIPTION
**Pull request by Rainchus**

### CODE REVIEW

File: src/code_a5dd0_len_114e0.c [hunk 1]
```diff
@@ -113,11 +122,71 @@ u32 get_entity_type(s32 index) {
     }
 }
 
-INCLUDE_ASM(s32, "code_a5dd0_len_114e0", delete_entity);
+void delete_entity(s32 entityIndex) {
+    Entity* entity;
+    Shadow* shadow;
+    EntityList** currentEntityListPtrTemp;
+
+    entity = get_entity_by_index(entityIndex);
```
Comment (nanaian) [inline src/code_a5dd0_len_114e0.c:130]: merge this with line 126

File: src/code_a5dd0_len_114e0.c [hunk 2]
```diff
@@ -113,11 +122,71 @@ u32 get_entity_type(s32 index) {
...
+        shadow = get_shadow_by_index(entity->shadowIndex);
+        shadow->flags |= 0x10000000;
+    }
+
+    currentEntityListPtrTemp = &gCurrentEntityListPtr;
+    heap_free((**currentEntityListPtrTemp)[entityIndex]);
+    (**currentEntityListPtrTemp)[entityIndex] = NULL;
+}
+
+//INCLUDE_ASM(s32, "code_a5dd0_len_114e0", func_8011085C);
```
Comment (nanaian) [inline src/code_a5dd0_len_114e0.c:178]: remove comment

File: src/code_a5dd0_len_114e0.c [hunk 3]
```diff
@@ -113,11 +122,71 @@ u32 get_entity_type(s32 index) {
...
+s32 func_8011085C(s32 entityIndex) {
+    s32 temp_s0;
+    s32 temp_v0;
+    Shadow* shadow;
+    ShadowList** currentShadowListPtr = &gCurrentShadowListPtr;
+
+    shadow = get_shadow_by_index(entityIndex);
```
Comment (nanaian) [inline src/code_a5dd0_len_114e0.c:185]: merge with line 182

File: src/code_a5dd0_len_114e0.c [hunk 4]
```diff
@@ -113,11 +122,71 @@ u32 get_entity_type(s32 index) {
...
+
+    currentEntityListPtrTemp = &gCurrentEntityListPtr;
+    heap_free((**currentEntityListPtrTemp)[entityIndex]);
+    (**currentEntityListPtrTemp)[entityIndex] = NULL;
+}
+
+//INCLUDE_ASM(s32, "code_a5dd0_len_114e0", func_8011085C);
+s32 func_8011085C(s32 entityIndex) {
+    s32 temp_s0;
+    s32 temp_v0;
```
Comment (nanaian) [inline src/code_a5dd0_len_114e0.c:181]: remove these two

**Review by nanaian — CHANGES_REQUESTED** [review]

**Review by nanaian — APPROVED** [review]

**Comment by ethteck** [comment]
gg!

**Pull request merged**